package.json

```diff
@@ -36,6 +36,7 @@
     "jsdoc": "^3.6.3",
     "medic-enketo-xslt": "^1.5.0",
     "mocha": "^6.1.4",
+    "moment": "^2.29.1",
```

Comment:
still needed?

Reply:
Core needs it 
```
ERROR in ./node_modules/cht-core-3-11/webapp/src/js/enketo/translator.js 1:15-32
Module not found: Error: Can't resolve 'moment' in '/home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/src/js/enketo'
resolve 'moment' in '/home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/src/js/enketo'
  Parsed request is a module
  using description file: /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/package.json (relative path: ./src/js/enketo)
    resolve as module
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/src/js/enketo/node_modules doesn't exist or is not a directory
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/src/js/node_modules doesn't exist or is not a directory
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/src/node_modules doesn't exist or is not a directory
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/webapp/node_modules doesn't exist or is not a directory
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/cht-core-3-11/node_modules doesn't exist or is not a directory
      /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules
        single file module
          using description file: /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/package.json (relative path: ./node_modules/moment)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/moment doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/moment.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/moment.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/moment.wasm doesn't exist
        /home/runner/work/medic-conf-test-harness/medic-conf-test-harness/node_modules/moment doesn't exist
      /home/runner/work/medic-conf-test-harness/node_modules doesn't exist or is not a directory
      /home/runner/work/node_modules doesn't exist or is not a directory
      /home/runner/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./ext/OpenrosaXpathEvaluatorBinding.js 7:17-73
 @ ./node_modules/enketo-core/src/js/Form-model.js 566:38-76
 @ ./node_modules/enketo-core/src/js/Form.js 3:16-41
 @ ./src/form-host/wireup.js 2:19-53
 @ ./src/form-host/index.js 2:19-38
```
https://github.com/medic/medic-conf-test-harness/runs/2883152895?check_suite_focus=true

Reply:
https://github.com/medic/cht-conf-test-harness/issues/126